tools/pfe-tools/esbuild.ts

```diff
@@ -66,6 +66,8 @@ const ALWAYS_EXCLUDE = [
   'pfe-styles',
 ];
 
+const exists = (filePath: string) => stat(filePath).then(() => true, () => false);
```

Comment:
Is using the `stat()` method better than fs.exists?

Reply:
yes, [fs.exists is deprecated](https://nodejs.org/dist/latest-v16.x/docs/api/fs.html#fsexistspath-callback)